### ISSUE TITLE
How to use template.html

### ISSUE THREAD

**Issue by Otto-AA** [preactjs-templates/default#32]
I want to add a bootstrap <link> to the html template, but when I try it webpack yields an error:
**✖ ERROR Template execution failed: ReferenceError: preact is not defined**

# Steps to reproduce
1. Install the default template (`preact create default my-project`)
2. Copy the [preact-cli template.html](https://github.com/preactjs/preact-cli/blob/master/packages/cli/lib/resources/template.html) to src/template.html
3. Run `preact watch --template src/template.html`

Does anyone know a way to achieve what I want to do without getting this error?

**Comment by infelipe1994** [comment]
Faced the same error today :(
Have you managed to figure something out?

**Comment by scottjbarr** [comment]
I had the same problem, but this seems to be fixed in 3.0.0-rc.7. 

Thanks for the help on the Slack channel from user kidqueb .

```
$ preact --version
preact, 3.0.0-rc.7
$ preact create default my-project
$ cd my-project
$ ls src/template.html # src/template.html was created by "preact create"
src/template.html
$ preact watch --template src/template.html # starts ok
```

**Comment by pnfcre** [comment]
Solution for me:

```diff
- <%preact.headEnd %>
+ <% preact.headEnd %>
```

I accidentally forgot a space between the `%` and `preact` 😄
